docs/sdk/ledger.md

```diff
@@ -0,0 +1,55 @@
+### Ledger API
+SDK API to access ledger and proofs data:
```

Comment:
I don't think this file belongs here at all! We will have hopefully multiple SDKs, just link to the right SDK repo(s). This should be in the go-doc of the Go sdk, and only there. or else, every time we change something in the SDK we have to to change here as well.

Reply:
Will move all folder to sdk documentation.

Reply:
Done